roles/kubernetes/cni/calico/tasks/main.yaml

```diff
@@ -23,7 +23,7 @@
 
 - name: Apply custom-resources
   ansible.builtin.shell: |
-    kubectl apply -f /tmp/custom-resources.yaml
+    kubectl create -f /tmp/custom-resources.yaml
   tags:
     - install
     - update
```

Comment:
Same comment as above need to break these out into `install` -> `kubectl create` and `update` -> `kubectl replace`